src/PointwiseFunctions/GeneralRelativity/Ricci.cpp

```diff
@@ -50,6 +51,27 @@ tnsr::aa<DataType, SpatialDim, Frame, Index> ricci_tensor(
                d_christoffel_2nd_kind);
   return result;
 }
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+void ricci_scalar(
+    const gsl::not_null<Scalar<DataType>*> ricci_scalar_result,
+    const tnsr::aa<DataType, SpatialDim, Frame, Index>& ricci_tensor,
+    const tnsr::AA<DataType, SpatialDim, Frame, Index>& inverse_metric) {
+  destructive_resize_components(ricci_scalar_result,
+                                get_size(get<0, 0>(inverse_metric)));
+
+  *ricci_scalar_result = trace(ricci_tensor, inverse_metric);
+}
+
+template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
+Scalar<DataType> ricci_scalar(
+    const tnsr::aa<DataType, SpatialDim, Frame, Index>& ricci_tensor,
+    const tnsr::AA<DataType, SpatialDim, Frame, Index>& inverse_metric) {
+  Scalar<DataType> ricci_scalar_result{get<0, 0>(inverse_metric)};
```

Comment:
use `make_with_value` to avoid copying data

Reply:
... or just make this function `return trace(...);` I guess :)